src/util/std_expr.h

```diff
@@ -2126,6 +2126,11 @@ inline not_exprt &to_not_expr(exprt &expr)
   return ret;
 }
 
+// Create a lexicographic less-than relation between two tuples of variables.
+// This is used in the implementation of multidimensional decreases clauses.
+exprt create_lexicographic_less_than(
+  const std::vector<symbol_exprt> &small,
+  const std::vector<symbol_exprt> &large);
 
```

Comment:
`std_expr.h` is the wrong place for this. Consider `expr_util.h`. Or leave it local with the code that uses it; `util/` is meant to be used when there is at least strong potential that there will be more than one user. This may happen one day, and then the function could move.

Reply:
I see. Thanks for your suggestion!